style-dictionary/visual-refresh/colors.ts

```diff
@@ -126,7 +126,7 @@ const tokens: StyleDictionary.ColorsDictionary = {
   colorBorderDividerSecondary: { light: '{colorGrey200}', dark: '{colorGrey700}' },
   colorBorderDropdownContainer: { light: '{colorGrey400}', dark: '{colorGrey550}' },
   colorBorderDropdownGroup: '{colorBorderDropdownItemDefault}',
-  colorBorderDropdownItemDefault: '{colorBorderDividerDefault}',
+  colorBorderDropdownItemDefault: 'transparent',
```

Comment:
Did not update width here because we have a lot of custom calculations that rely on border widths.